tensorflow/python/kernel_tests/math_ops/reduction_ops_test.py

```diff
@@ -280,6 +280,46 @@ def testFloat32(self):
             tf_out_sum_xz, tf_out_sum_y = self.evaluate([tf_sum_xz, tf_sum_y])
           self.assertAllClose(sum_y, tf_out_sum_y)
           self.assertAllClose(sum_xz, tf_out_sum_xz)
+  
```

Comment:
If this is just a copy of the Float32 test, can we instead make that parameterized on the dtype?

Reply:
The updated test is added and the conflicts are also resolved.